### PR TITLE
Update: Java Math.sqrt method

### PR DESCRIPTION
**Pull request by theflametrooper**
Original Article by @nag786 
Original PR: https://github.com/FreeCodeCamp/wiki/pull/799
Implemented changes suggested by @alayek

### CODE REVIEW

**Comment by Rafase282** [comment]
# LGTM

File: Java-Math.sqrt.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+ }
+```
+Output:
+```
+Math Square root value: 3.0
+Math Square root value: 49.565108695532984
+Math Square root value: NaN
+```
+
```
Comment (koustuvsinha) [inline Java-Math.sqrt.md]: Add a :rocket: [REPL.it](repl.it) snippet

**Comment by koustuvsinha** [comment]
@theflametrooper add the snippet, other than that, LGTM :+1:

File: Java-Math.sqrt.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  double a = Math.sqrt(9); //calculates square root of 9
+  double b = Math.sqrt(2456.7); //calculates square root of 2456.7
+  double c = Math.sqrt(-4568); //calculates square root of -4568
+
+  System.out.println("Math Square root value :" + a);
+  System.out.println("Math Square root value :" + b);
+  System.out.println("Math Square root value :" + c);
+
+ }
```
Comment (koustuvsinha) [inline Java-Math.sqrt.md]: close the public class Math { brace }. thanks to @theflametrooper for pointing it out :stuck_out_tongue_closed_eyes:

**Pull request merged**